toponetx/algorithms/distance_measures.py

```diff
@@ -0,0 +1,196 @@
+"""Module to distance measures on topological domains."""
+from collections.abc import Collection, Hashable, Iterable, Iterator
+from warnings import warn
+
+import networkx as nx
+import numpy as np
+
+from toponetx.classes.cell_complex import CellComplex
+from toponetx.classes.colored_hypergraph import ColoredHyperGraph
+from toponetx.classes.combinatorial_complex import CombinatorialComplex
+from toponetx.classes.simplicial_complex import SimplicialComplex
+from toponetx.exception import TopoNetXError
+
+__all__ = ["node_diameters", "cell_diameters", "diameter", "cell_diameter"]
+
+
+def node_diameters(complex):
+    """Return the node diameters of the connected components in cell complex.
+
+    Returns
+    -------
+    diameters : list
+        List of the diameters of the s-components.
+    components : list
+        List of the s-component nodes.
+
+    Example
```

Comment:
NumPy's documentation style dictates that this headline should read Examples even if there is only one (same throughout this file).